letsencrypt-apache/letsencrypt_apache/configurator.py

```diff
@@ -1,4 +1,6 @@
 """Apache Configuration based off of Augeas Configurator."""
+# pylint: disable=too-many-lines
```

Comment:
Yeah, some further organizational edits need to be made.  Though, I will save it for a followup. :/